rubies/ruby/CVE-2020-10663.yml

```diff
@@ -0,0 +1,29 @@
+---
+engine: ruby
+cve: 2020-10663
+url: https://www.ruby-lang.org/en/news/2020/03/19/json-dos-cve-2020-10663/
+title: Unsafe Object Creation Vulnerability in JSON (Additional fix)
+date: 2020-03-19
+description: |
+  When parsing certain JSON documents, the json gem (including the one bundled
+  with Ruby) can be coerced into creating arbitrary objects in the target
+  system.
+
+  This is the same issue as CVE-2013-0269. The previous fix was incomplete, which
+  addressed `JSON.parse(user_input)`, but didn’t address some other styles of JSON
+  parsing including `JSON(user_input)` and `JSON.parse(user_input, nil)`.
+
+  See CVE-2013-0269 in detail. Note that the issue was exploitable to cause a
+  Denial of Service by creating many garbage-uncollectable Symbol objects, but
+  this kind of attack is no longer valid because Symbol objects are now
+  garbage-collectable. However, creating arbitrary objects may cause severe
+  security consequences depending upon the application code.
+
+  Please update the json gem to version 2.3.0 or later. You can use `gem update
+  json` to update it. If you are using bundler, please add `gem "json", ">= 2.3.0"`
+  to your Gemfile
+patched_versions:
+  - "~> 2.4.10"
+  - "~> 2.5.8"
+  - "~> 2.6.6"
+  - "> 2.7.1"
```

Comment:
`>=`, as 2.7.1 should be included.

Reply:
urgh yes, it seems I had brain fade when I made this. Will fix now.